### ISSUE TITLE
[Bug] GRASS GIS 8.0 doesn't launch

### ISSUE THREAD

**Issue by tmszi** [OSGeo/grass#1911]
**Describe the bug**
GRASS GIS 8.0 doesn't launch.

**To Reproduce**

1. Download [Standalone Win GRASS GIS 8.0 installer](https://wingrass.fsv.cvut.cz/grass80/x86_64/WinGRASS-8.0.dev-257-Setup-x86_64.exe)
2. Install GRASS GIS
3. Try launch GRASS GIS 8.0
4.  Quickly appear and disappear GRASS GIS session Command Prompt without error message

**Expected behavior**
Launch GRASS GIS.

**Screenshots**

![mswin_grass80_py_startup_file](https://user-images.githubusercontent.com/50632337/135665004-91551f9f-f44f-442a-8a19-e6833074e8c3.png)

**System description (please complete the following information):**
- MS Windows
- Standalone [WinGRASS-8.0.dev-257-Setup-x86_64.exe installer](https://wingrass.fsv.cvut.cz/grass80/x86_64/WinGRASS-8.0.dev-257-Setup-x86_64.exe)

**Additional context**
GISBASE directory `C:\Program Files\GRASS GIS 8.0\etc\` contains 2 startup GRASS GIS Python files `grass80.py` (which is empty) and `grass.py`. Startup command inside `grass80.bat` file point to` "%GRASS_PYTHON%" "%GISBASE%\etc\grass80.py" %*`.

**Comment by neteler** [comment]
Does this persist with the latest version?
https://wingrass.fsv.cvut.cz/grass80/x86_64/

**Comment by dnewcomb** [comment]
Yes, still a problem with the binary installer from 10/13

**Comment by tmszi** [comment]
Yes, it still doesn't work (Standalone Win GRASS GIS installer 2021-10-13).

**Comment by dnewcomb** [comment]
Apologies for my post on 12th, it was meant for 7.8.6-1.  However, the nightly standalone installer for 8.0 for 11/14 https://wingrass.fsv.cvut.cz/grass80/x86_64/WinGRASS-8.0.dev-7be7cf9b0-2-Setup.exe has the same issue as 7.8.6-1.
There are no read permissions for the c:\program files\grass 8.0\etc\grass8.0.py for standard users
This may be a different bug than the original.
When trying to launch grass as a standard user ( dlocal in this case)  

![GRASS8bug](https://user-images.githubusercontent.com/2164971/142092534-7e0ebe2d-2d9e-4cdb-8f12-83a0310b50ab.png)

dlocal has no read access to grass80.py

![GRASS8bug2](https://user-images.githubusercontent.com/2164971/142092539-a95aba41-8717-4f6b-9ac1-29d2215a1b30.png)
and therefore cannot run it.

**Comment by tmszi** [comment]
I was successfull with https://wingrass.fsv.cvut.cz/grass80/x86_64/WinGRASS-8.0.dev-7be7cf9b0-2-Setup.exe.

But another bug appear (GRASS GIS  CMD launch, but wxGUI doesn't launch).

```
C:\Users\IEUser>ERROR: wxGUI requires wxPython. DLL load failed while importing _core: The specified module could not be found.
You can still use GRASS GIS modules in the command line or in Python.
```

```
C:\Users\IEUser>"C:\Program Files\GRASS GIS 8.0\Python39\python.exe" -c "import wx"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Program Files\GRASS GIS 8.0\Python39\lib\site-packages\wx\__init__.py", line 17, in <module>
    from wx.core import *
  File "C:\Program Files\GRASS GIS 8.0\Python39\lib\site-packages\wx\core.py", line 12, in <module>
    from ._core import *
ImportError: DLL load failed while importing _core: The specified module could not be found.
```

**Comment by hellik** [comment]
winGRASS 8.0dev starts without any problem in a Windows 10 Home laptop

**Comment by tmszi** [comment]
Closed by #1933.

**Comment by tmszi** [comment]
Closed by #1933.
